### PR TITLE
Change ws_db to ws_agent

### PR DESCRIPTION
**Pull request by mharcar**

### CODE REVIEW

File: src/Monitor/Agents.vue [hunk 1]
```diff
@@ -57,8 +57,8 @@ export default {
       }
     },
     query: async function() {
-      this.groups = await this.$agent.p_mfa('ws_db_agent_group', 'get')
-      this.agents = await this.$agent.p_mfa('ws_admin', 'agents', ['all'])
+      this.groups = await this.$agent.p_mfa('ws_agent_agent_group', 'get')
```
Comment (jamhed) [inline src/Monitor/Agents.vue]: Should be ws_agent, agent_groups

File: src/Monitor/Agents.vue [hunk 2]
```diff
@@ -57,8 +57,8 @@ export default {
...
+      this.agents = await this.$agent.p_mfa('ws_agent', 'agents', ['all'])
```
Comment (jamhed) [inline src/Monitor/Agents.vue]: Should not have ['all']

File: src/Monitor/Agents.vue (hunk 1 above)
Comment (mharcar) [reply src/Monitor/Agents.vue]: p_mfa('ws_agent', 'agent_group', 'get') ?

File: src/Monitor/Inqueues.vue [hunk 3]
```diff
@@ -53,8 +53,8 @@ export default {
       return info
     },
     query: async function() {
-      this.queues = await this.$agent.p_mfa('ws_db_queue', 'get', [])
-      this.inqueues = await this.$agent.p_mfa('ws_admin', 'inqueues', ['all'])
+      this.queues = await this.$agent.p_mfa('ws_agent_queue', 'get', [])
+      this.inqueues = await this.$agent.p_mfa('ws_agent', 'inqueues', ['all'])
```
Comment (jamhed) [inline src/Monitor/Inqueues.vue]: Shoud not have ['all']

File: src/Monitor/Widget/AgentMonitor.vue [hunk 4]
```diff
@@ -394,9 +394,9 @@ export default {
         this.updateStats()
     },
     query: async function() {
-      this.clients = await this.$agent.p_mfa('ws_db_client', 'get')
+      this.clients = await this.$agent.p_mfa('ws_agent_client', 'get')
```
Comment (jamhed) [inline src/Monitor/Widget/AgentMonitor.vue]: should be (ws_agent, clients)

File: src/Monitor/Widget/AgentMonitor.vue [hunk 5]
```diff
@@ -394,9 +394,9 @@ export default {
...
       this.clients.unshift({ name:"Any Customers" })
-      this.tags = await this.$agent.p_mfa('ws_db_tag', 'get')
+      this.tags = await this.$agent.p_mfa('ws_agent_tag', 'get')
```
Comment (jamhed) [inline src/Monitor/Widget/AgentMonitor.vue]: should be ws_agent, tags

File: src/Monitor/Widget/InqueuesMonitor.vue [hunk 6]
```diff
@@ -129,13 +129,13 @@ export default {
   },
   methods: {
     query: async function() {
-      this.clients = await this.$agent.p_mfa('ws_db_client', 'get')
+      this.clients = await this.$agent.p_mfa('ws_agent_client', 'get')
```
Comment (jamhed) [inline src/Monitor/Widget/InqueuesMonitor.vue]: And so on

File: src/Monitor/Agents.vue (hunk 1 above)
Comment (jamhed) [reply src/Monitor/Agents.vue]: p_mfa('ws_agent', 'agent_groups')

File: src/Monitor/Agents.vue [hunk 7]
```diff
@@ -57,8 +57,8 @@ export default {
       }
     },
     query: async function() {
-      this.groups = await this.$agent.p_mfa('ws_agent_agent_group', 'get')
-      this.agents = await this.$agent.p_mfa('ws_agent', 'agents', ['all'])
+      this.groups = await this.$agent.p_mfa('ws_agent', 'agent_group')
```
Comment (jamhed) [inline src/Monitor/Agents.vue]: agent_groups

File: src/Monitor/Inqueues.vue [hunk 8]
```diff
@@ -53,8 +53,8 @@ export default {
       return info
     },
     query: async function() {
-      this.queues = await this.$agent.p_mfa('ws_agent_queue', 'get', [])
-      this.inqueues = await this.$agent.p_mfa('ws_agent', 'inqueues', ['all'])
+      this.queues = await this.$agent.p_mfa('ws_agent', 'queue')
```
Comment (jamhed) [inline src/Monitor/Inqueues.vue]: queues

File: src/Monitor/Widget/AgentMonitor.vue [hunk 9]
```diff
@@ -394,9 +394,9 @@ export default {
         this.updateStats()
     },
     query: async function() {
-      this.clients = await this.$agent.p_mfa('ws_agent_client', 'get')
+      this.clients = await this.$agent.p_mfa('ws_agent', 'client')
```
Comment (jamhed) [inline src/Monitor/Widget/AgentMonitor.vue]: clients

File: src/Monitor/Widget/AgentMonitor.vue [hunk 10]
```diff
@@ -394,9 +394,9 @@ export default {
...
       this.clients.unshift({ name:"Any Customers" })
-      this.tags = await this.$agent.p_mfa('ws_agent_tag', 'get')
+      this.tags = await this.$agent.p_mfa('ws_agent', 'tag')
```
Comment (jamhed) [inline src/Monitor/Widget/AgentMonitor.vue]: tags

File: src/Monitor/Widget/InqueuesMonitor.vue [hunk 11]
```diff
@@ -129,13 +129,13 @@ export default {
...
   methods: {
     query: async function() {
-      this.clients = await this.$agent.p_mfa('ws_agent_client', 'get')
+      this.clients = await this.$agent.p_mfa('ws_agent', 'client')
       this.clients.unshift({ name:"Any Customers" })
-      this.queues = await this.$agent.p_mfa('ws_agent_queue', 'get', [])
+      this.queues = await this.$agent.p_mfa('ws_agent', 'queue')
       this.queues.unshift({ name:"Any Queue" })
-      this.lines = await this.$agent.p_mfa('ws_agent_line_in', 'get')
+      this.lines = await this.$agent.p_mfa('ws_agent', 'line_in')
```
Comment (jamhed) [inline src/Monitor/Widget/InqueuesMonitor.vue]: line_ins

**Pull request merged**